app/styles/ui/changes/_commit-message.scss

```
@@ -202,3 +223,7 @@
     }
   }
 }
+
+.length-hint-tooltip {
+  max-width: none !important; // The i
```

Looks like the comment here got cut off?
```suggestion
  max-width: none !important; // The i
```

Secondarily, if we still need this max-width we should move it to the `_tooltips.scss` declaration for this class which should also negate the need for the `!important` modifier.